src/compile/mark/valueref.ts

```diff
@@ -85,9 +88,11 @@ export function midPoint(channel: Channel, channelDef: ChannelDef, scaleName: st
   if (channelDef) {
     /* istanbul ignore else */
     if (isFieldDef(channelDef)) {
-      if (scale.type === 'bin-linear') {
-        return binMidSignal(channelDef, scaleName);
-      } else if (scale.type === 'bin-ordinal') {
+      if (isBinScale(scale.type)) {
+        // Use middle only for x an y to place marks in the center between start and end of the bin range. 
```

Comment:
Wanna add that to comment with a bit better description than "as we saw today"? :p